src/control/server/engine/config_test.go

```diff
@@ -1155,23 +1163,39 @@ func TestConfig_ValidateAndAdjustPMDKEnvVar(t *testing.T) {
 				storage.NewTierConfig().
 					WithStorageClass("dcpm"),
 			).WithEnvVarPMemObjSdsAtCreate(1),
-			expErr: errors.New("env_var PMEMOBJ_CONF should NOT be set to sds.at_create=? for 'dcpm' storage class"),
+			expErr: errors.New("env_var PMEMOBJ_CONF should NOT contain " +
+				"'sds.at_create=?' for DCPM storage class"),
 		},
 		"config for DCPM with forced sds.at_create (0) should fail": {
 			cfg: validConfig().WithStorage(
 				storage.NewTierConfig().
 					WithStorageClass("dcpm"),
 			).WithEnvVarPMemObjSdsAtCreate(0),
-			expErr: errors.New("env_var PMEMOBJ_CONF should NOT be set to sds.at_create=? for 'dcpm' storage class"),
+			expErr: errors.New("env_var PMEMOBJ_CONF should NOT contain " +
+				"'sds.at_create=?' for DCPM storage class"),
 		},
-		"Valid config for ram should not fail": {
+		"valid config for ram should not fail": {
 			cfg: validConfig().WithStorage(
 				storage.NewTierConfig().
 					WithStorageClass("ram"),
 			).WithProperEnvVarForPMDK(),
 			expSds_at_create: "sds.at_create=0",
 		},
-		"Valid config with default ULT stack size for ram should not fail": {
+		"config for ram should should be updated": {
+			cfg: validConfig().WithStorage(
+				storage.NewTierConfig().
+					WithStorageClass("ram"),
+			).WithoutEnvVarForPMDK(),
+			expSds_at_create: "sds.at_create=0",
+		},
+		"config for ram with PMEMOBJ_CONF should should be updated": {
+			cfg: validConfig().WithStorage(
+				storage.NewTierConfig().
+					WithStorageClass("ram"),
+			).WithoutEnvVarForPMDK().WithEnvVars("PMEMOBJ_CONF=foo_bar"),
+			expSds_at_create: "foo_bar;sds.at_create=0",
```

Comment:
we don't use underscores in go variable names

Reply:
Done